connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/PlanGenerationTestSuite.scala

```diff
@@ -3017,6 +3017,12 @@ class PlanGenerationTestSuite
     simple.groupBy(Column("id")).pivot("a").agg(functions.count(Column("b")))
   }
 
+  test("SPARK-46048: groupingSets") {
```

Comment:
The the golden files name generated after adding the Jira Id looks a bit strange

Reply:
Yeah, probably let me rename the test ..